x-pack/plugins/observability_solution/infra/public/pages/metrics/metric_detail/asset_detail_page.tsx

```diff
@@ -43,7 +43,7 @@ export const AssetDetailPage = () => {
     <AssetDetails
       assetId={nodeId}
       assetType={nodeType}
-      tabs={commonFlyoutTabs}
+      tabs={assetDetailsTabs(nodeType)}
```

Comment:
Should we rename this function?
```suggestion
      tabs={getAssetDetailsTabs(nodeType)}
```